src/view.js

```diff
@@ -0,0 +1,51 @@
+function runView() {
+  const calc = document.querySelector('.calc');
+
+  const calcScreen = document.createElement('div');
```

Comment:
```suggestion
  const createComponent = ({ tagName, className, parentNode }) => {
    const node = document.createElement(tagName);
    node.className = className;
    parentNode.append(node);
    return node;
  }
```

Ты выносишь повторяещие элементы и можешь переисользовать такие функции, таким образом сократишь твой код